plugins/evaluation-plugin/core/src/com/intellij/cce/processor/CallCompletionProcessor.kt

```diff
@@ -43,6 +44,34 @@ class CallCompletionProcessor(private val text: String,
   }
 
   private fun processToken(token: CodeToken) {
+    when(token.properties.tokenType) {
+      TypeProperty.VARIABLE_DECLARATION -> processVariableDeclaration(token)
+      else -> processCompletion(token)
+    }
+  }
+
+  private fun processVariableDeclaration(token: CodeToken) {
+    if (!checkFilters(token)) return
+
+    when (strategy.context) {
+      CompletionContext.ALL -> prepareAllContext(token)
+      CompletionContext.PREVIOUS -> preparePreviousContext(token)
+    }
+
+    if (strategy.emulateUser) {
+      addAction(EmulateUserSession(token.text, token.properties))
+    } else {
+      val prefix = token.text[0].toString()
```

Comment:
suggest print something like `variableToRename` instead of variable prefix to make it more explicit.